app/scripts/lib/account-tracker.test.js

```diff
@@ -39,6 +37,10 @@ const mockAccounts = {
   },
 };
 
+function flushPromises() {
+  return new Promise(jest.requireActual('timers').setImmediate);
+}
+
```

Comment:
This is now a duplicate (also used in detect-tokens.js) we should pull it out into a helper file with some comments about what its doing.

Reply:
done https://github.com/MetaMask/metamask-extension/pull/22217/commits/873f6671e2d4be7af44ecb35fd7e13fbf630e1ad

Reply:
linter doesn't like `jest` being used in a non *.test file. We can pass it in as an arg to flushPromises? seems weird though

The docs actually mention this function in it's entirety
https://github.com/MetaMask/metamask-extension/blob/develop/docs/testing.md?plain=1#L160

Not sure what the best approach is here

Reply:
fixed lint issue here https://github.com/MetaMask/metamask-extension/pull/22217/commits/e24e829eef6c29832043fd592144d2852588d277